plugins/chunter-resources/src/components/ChannelScrollView.svelte

```diff
@@ -131,8 +132,23 @@
       }
     })
 
+  let isPageHidden = false
+  let lastMsgBeforeFreeze: Ref<ActivityMessage> | undefined = undefined
+
+  document.addEventListener('visibilitychange', () => {
```

Comment:
Please use svelte document event listener registration or unregistered it after used.

Reply:
Done